ui/src/components/payment/payment-dialog/PaymentDialog.tsx

```diff
@@ -130,15 +139,29 @@ export default class PaymentDialog extends Component<PaymentProps, PaymentState>
     };
 
     private fetchMonthsCountItems = () => {
-        return ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"];
+        let months = [];
+        for (let i = 1; i < 10; i++) {
+            months.push(i.toString())
+        }
+        return months;
+    };
+
+    private populateRooms = async () => {
+        const dormitory = this.state.paymentDetails.dormitory;
+        if (ObjectUtils.checkNotNull(dormitory)) {
+            const rooms = await this.dormitoryService.fetchDormitoryRooms(dormitory.id);
+            this.setState({
+                rooms: rooms
+            })
+        }
     };
 
-    private fetchRoomMenuItems = () => {
-        return ["1", "2", "3", "4", "5"];
+    private fetchRooms = () => {
+        return this.state.rooms.map(room => room.title);
```

Comment:
We can a little bit rework it.

We can pass fully objects to `Autocomplete`, but inside of `Autocomplete` override function `getOptionLabel` (this function responsible for `option` displaying). The same we can apply for `Dormitories`.

Reply:
i think i got an idea, i'll try that